resources/i18n/en.json

```diff
@@ -218,8 +219,9 @@
       "EnvConfigWillDisappear": "Any unsaved environment variable and value will be disappeared.",
       "SharedMemoryAbbr": "SHMEM",
       "GPUSlot": "Slot",
-      "FolderAlias": "Alias",
+      "FolderAlias": "Path & Alias",
       "FolderAliasOverlapping": "Aliases overlap with other names that already exist",
+      "DescFolderAlias":"<p>1. If you fill out this input, folder name will change to alias when it mount<br/>2. Start with /: absolute path<br/>3. Not start with / or blank: sub path of /home/work<br/>4. Don't use alias same as default system folder of linux.<br/>5. Please check final path in 'Mounted folders' below.</p>",
```

Comment:
I think this is important information, so It should be exposed automatically when the active carousel page is on the "mount page"(which is, page 2). For now, user can see the guide only if they click the info icon.
Also, Let's use a shorter but more powerful statement. Let's say, the statement "If you fill out this input, folder name will change to alias when it mount", seems too descriptive. I'd like to suggest "Folder name will be changed to the alias when it mounts." the other statements seem to be Okay, but It would be appreciated to polish them up.
Lastly, a number list would be appropriate when order matters for each instruction. I don't think these numbers are meaningful in this instruction. How about using a bullet list instead of a number list?

Reply:
Please remove HTML tag from language file. (it should be on the .ts file). Some parts have inline HTML tags in the language resource, however they were inevitable.

Break the line into 5 resources.

Reply:
@inureyes Hmm, we've been using HTML tags in here. Is there any specific reason to get rid of tags? If we do split the resource into five keys and values, it's a bit over-descriptive.

Reply:
For the reasons described above and for the unity with other '_helpDescription' in the code, only the <p> tag was used.